src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnum.java

```diff
@@ -104,13 +111,12 @@ public String getColumnValue(final VariantContext variant) {
                 } else {
                     outValue = outNotAlleleSpecificAndOld;
                 }
-                // Spread MQ accross multiple alleles.
-                //TODO: check how hail does this and change this to do the same thing
+
                 double mq = Double.parseDouble(outValue);
                 if (variant.getAlleles().size() == 3) {
-                    outNotAlleleSpecific = (int) mq / 2 + VCFConstants.PHASED + (int) mq / 2;
+                    outNotAlleleSpecific = (int) mq + VCFConstants.PHASED + (int) mq;
                 } else if (variant.getAlleles().size() == 4) {
-                    outNotAlleleSpecific = (int) mq / 3 + VCFConstants.PHASED + (int) mq / 3 + VCFConstants.PHASED + (int) mq / 3;
+                    outNotAlleleSpecific = (int) mq + VCFConstants.PHASED + (int) mq + VCFConstants.PHASED + (int) mq;
                 } else {
                     throw new UserException("Expected diploid sample to either have 3 alleles (ref, alt, non-ref) or 4 alleles (ref, alt 1, alt 2, non-ref)");
```

Comment:
I would have done this as an initial check and then a loop over the number of alleles rather than hard coding 3 and 4 in the conditionals, but maybe that's just six one way half a dozen the other.

Reply:
yeah -- I don't disagree that this class could use some further refactoring